sagemaker-python-sdk/tensorflow_serving_using_elastic_inference_with_your_own_model/tensorflow_serving_pretrained_model_elastic_inference.ipynb

```diff
@@ -167,7 +163,7 @@
     "import numpy as np\n",
     "random_input = np.random.rand(1, 1, 3, 3)\n",
     "\n",
-    "prediction = predictor.predict({'input': random_input.tolist()})\n",
+    "prediction = predictor.predict({'inputs': random_input.tolist()})\n",
```

Comment:
Can you explain why the 'inputs' keyword is provided here?

Reply:
TFS container checks if input data is labeled as 'instances/inputs/examples' (https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/container/sagemaker/tensorflow-serving.js#L131). If yes, it will pass the data as it is. Otherwise, for example, if the label is 'input', it prepends an 'instances' to the data (https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/container/sagemaker/tensorflow-serving.js#L139).

Reply:
We should mention that in the notebook, as it seems this functionality is TFS container specific.

Reply:
Actually it is a functionality of TFS: https://www.tensorflow.org/tfx/serving/api_rest#specifying_input_tensors_in_row_format